cron/projects.txt

```diff
@@ -1,2314 +1,2314 @@
+github.com/Elgg/Elgg,
+github.com/EventStore/EventStore,
+github.com/FFmpeg/FFmpeg,
+github.com/FRRouting/frr,
+github.com/FakeItEasy/FakeItEasy,
+github.com/FasterXML/jackson-core,
+github.com/FasterXML/jackson-databind,
+github.com/Flexget/Flexget,
+github.com/FluentValidation/FluentValidation,
+github.com/Fody/Fody,
+github.com/FreeCAD/FreeCAD,
+github.com/FreeRADIUS/freeradius-server,
+github.com/FreeRDP/FreeRDP,
+github.com/FreshRSS/FreshRSS,
+github.com/FriendsOfPHP/PHP-CS-Fixer,
+github.com/FriendsOfSymfony/FOSRestBundle,
+github.com/Frogging-Family/wine-tkg-git,
+github.com/Froxlor/Froxlor,
+github.com/Fyrd/caniuse,
+github.com/GNOME/gimp,
+github.com/GNOME/glib,
+github.com/GNOME/gnome-shell,
+github.com/GNOME/gtk,
+github.com/GNOME/librsvg,
+github.com/GameServerManagers/LinuxGSM,
+github.com/Geal/nom,
+github.com/GenericMappingTools/gmt,
+github.com/Genymobile/scrcpy,
+github.com/GitTools/GitVersion,
+github.com/Gnucash/gnucash,
+github.com/GoogleContainerTools/skaffold,
+github.com/Graylog2/graylog2-server,
+github.com/Hack23/cia,
+github.com/Hack23/securityfixerbot,
+github.com/Hack23/sonar-cloudformation-plugin,
+github.com/Hacker0x01/react-datepicker,
+github.com/HandBrake/HandBrake,
+github.com/HerculesWS/Hercules,
+github.com/Homebrew/brew,
+github.com/Homebrew/homebrew-cask,
+github.com/Homebrew/homebrew-cask-fonts,
+github.com/Homebrew/homebrew-cask-versions,
+github.com/Homebrew/homebrew-core,
+github.com/Homebrew/install,
+github.com/Homebrew/linuxbrew-core,
+github.com/HospitalRun/hospitalrun-frontend,
+github.com/Humanizr/Humanizer,
+github.com/HypothesisWorks/hypothesis,
+github.com/ILIAS-eLearning/ILIAS,
+github.com/IQSS/dataverse,
+github.com/Icinga/icinga2,
+github.com/IdentityServer/IdentityServer4,
+github.com/IgniteUI/igniteui-angular,
+github.com/ImageMagick/ImageMagick,
+github.com/InsightSoftwareConsortium/ITK,
+github.com/IntelRealSense/realsense-ros,
+github.com/JabRef/jabref,
+github.com/Jackett/Jackett,
+github.com/JamesNK/Newtonsoft.Json,
+github.com/JasperFx/marten,
+github.com/JeffreySu/WeiXinMPSDK,
+github.com/Jermolene/TiddlyWiki5,
+github.com/JetBrains/intellij-scala,
+github.com/JetBrains/resharper-unity,
+github.com/JohnSully/KeyDB,
+github.com/JoshClose/CsvHelper,
+github.com/JustArchiNET/ArchiSteamFarm,
+github.com/KSP-CKAN/CKAN,
+github.com/Katello/katello,
+github.com/KhronosGroup/glslang,
+github.com/Kitware/CMake,
+github.com/KnpLabs/php-github-api,
+github.com/Koenkk/zigbee-herdsman-converters,
+github.com/Koenkk/zigbee2mqtt,
+github.com/KratosMultiphysics/Kratos,
+github.com/Kunena/Kunena-Forum,
+github.com/Kunzisoft/KeePassDX,
+github.com/LCTT/TranslateProject,
+github.com/LMMS/lmms,
+github.com/Laravel-Backpack/CRUD,
+github.com/Laravel-Lang/lang,
+github.com/LemmyNet/lemmy,
+github.com/LessWrong2/Lesswrong2,
+github.com/Lienol/openwrt,
+github.com/LimeSurvey/LimeSurvey,
+github.com/LonamiWebs/Telethon,
+github.com/LuaJIT/LuaJIT,"project=envoy,dependency=true"
```

Comment:
@asraa FYI... The projects.txt is now sorted and also provides metadata in the output to track.